core/src/test/java/org/lint/azzert/AssertTestNGStyle.java

```diff
@@ -0,0 +1,34 @@
+package org.lint.azzert;
```

Comment:
Did you mean to test methods with parameters as well? Handling parameters would certainly enrich the output and help differentiate between overloaded methods.